functions-and-operators/miscellaneous-functions.md

```diff
@@ -23,15 +24,386 @@ TiDB supports most of the [miscellaneous functions](https://dev.mysql.com/doc/re
+```
+
+```
++-------------------------------------------+
+| IS_IPV4_COMPAT(INET6_ATON('::127.0.0.1')) |
++-------------------------------------------+
+|                                         1 |
++-------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+### IS_IPV4_MAPPED()
+
+Tests if the argument is a IPv4-mapped address.
+
+```sql
+SELECT IS_IPV4_MAPPED(INET6_ATON('::ffff:127.0.0.1'));
+```
+
+```
++------------------------------------------------+
+| IS_IPV4_MAPPED(INET6_ATON('::ffff:127.0.0.1')) |
++------------------------------------------------+
+|                                              1 |
++------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+### IS_IPV6()
+
+Tests if the argument is an IPv6 address.
+
+```sql
+SELECT IS_IPV6('::1');
+```
+
+```
++----------------+
+| IS_IPV6('::1') |
++----------------+
+|              1 |
++----------------+
+1 row in set (0.00 sec)
+```
+### IS_UUID()
+
+Tests if the argument is a UUID.
+
+```sql
+SELECT IS_UUID('eb48c08c-eb71-11ee-bacf-5405db7aad56');
+```
+
+```
++-------------------------------------------------+
+| IS_UUID('eb48c08c-eb71-11ee-bacf-5405db7aad56') |
++-------------------------------------------------+
+|                                               1 |
++-------------------------------------------------+
+1 row in set (0.00 sec)
+
+```
+
+### NAME_CONST()
+
+This is an function for naming columns. It is recommended to use column aliases instead.
+
+```sql
+SELECT NAME_CONST('column name', 'value') UNION ALL SELECT 'another value';
+```
+
+```
++---------------+
+| column name   |
++---------------+
+| another value |
+| value         |
++---------------+
+2 rows in set (0.00 sec)
+```
+
+The statement above uses `NAME_CONST()` and the statement below uses the recommended way of aliasing columns.
+
+```sql
+SELECT 'value' AS 'column name' UNION ALL SELECT 'another value';
+```
+
+```
++---------------+
+| column name   |
++---------------+
+| value         |
+| another value |
++---------------+
+2 rows in set (0.00 sec)
+```
+
+### SLEEP()
+
+Sleep for a number of seconds.
+
+```sql
+SELECT SLEEP(1.5);
+```
+
+```
++------------+
+| SLEEP(1.5) |
++------------+
+|          0 |
++------------+
+1 row in set (1.50 sec)
+```
+
+### UUID()
+
+Gets a UUID v1 as defined in [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122).
+
+```sql
+SELECT UUID();
+```
+
+```
++--------------------------------------+
+| UUID()                               |
++--------------------------------------+
+| cb4d5ae6-eb6b-11ee-bacf-5405db7aad56 |
++--------------------------------------+
+1 row in set (0.00 sec)
+```
+
+See also the [best practices for UUID](/best-practices/uuid.md)
+
+### UUID_TO_BIN
+
+See [BIN_TO_UUID()](#bin_to_uuid).
+
+### VALUES()
+
+This function is used to specify values in the `ON DUPLICATE KEY UPDATE` part of the [`INSERT`](/sql-statements/sql-statement-insert.md) statement.
```

Comment:
It is not clear to which value this refers in IODKU, maybe clarify it? I would assume that it refers to the value for the specific column, that was tried to be inserted? In this case the first `2`? If the example had more unique values then it would be easy to refer to the specific value.
And if my assumption is correct, then maybe another tweak of the example would be `... ON DUPLICATE KEY UPDATE c1=c1+VALUES(id)` which also works for a multi-row insert, which is one of the use cases.